api/base.proto

```diff
@@ -227,3 +227,11 @@ message TransportSocket {
   // See the supported transport socket implementations for further documentation.
   google.protobuf.Struct config = 2;
 }
+
+// Percent, typically used to specify things like target sampling percentages among tracing requests
+// (as in, e.g., :ref:`HTTP Connection Manager tracing
+// <envoy_api_field_filter.network.HttpConnectionManager.tracing>`).
+message Percent {
+  // The percent, a float between 0 and 1.
+  float value = 1 [(validate.rules).float = {gte: 0, lte: 1}];
```

Comment:
I would also say that I don't feel very strongly about 0.0-1.0 if people prefer 0.0-100.0.

Reply:
I prefer ~0.0-~1.0, which I think most people would say is more intuitive.

I would consider replacing 1.0 with some other number (_e.g._, 100) if we were planning on users needing that extra resolution between numbers. I do not get the sense they will, though.

Reply:
If the message is called percent, the value range should be [0, 100]. That is how most people interpret English words.

Reply:
SGTM, I think per above we should switch to `RoundedPercent`, double, 0.0-100.0